scripts/popup.js

```diff
@@ -158,14 +165,31 @@ function social_share(eventObj) {
     }
     var open_url = ''
     if (isNotExcludedUrl(url)) { // Prevents sharing some unnecessary page
-      if (id.includes('fb')) {
-        open_url = 'https://www.facebook.com/sharer/sharer.php?u=' + sharing_url // Share the wayback machine's overview of the URL
-      } else if (id.includes('twit')) {
-        open_url = 'https://twitter.com/intent/tweet?url=' + sharing_url
-      } else if (id.includes('linkedin')) {
-        open_url = 'https://www.linkedin.com/shareArticle?url=' + sharing_url
+      if (id.includes('copy')) {
+        const toastMsg = document.getElementsByClassName('toast')
+        navigator.clipboard.writeText(sharing_url).then(function() {
+          toastMsg[0].setAttribute('id', 'toast-success')
+          toastMsg[0].innerText = 'Page link copied.'
+          showToast(toastMsg[0])
+        }, function() {
+          toastMsg[0].setAttribute('id', 'toast-fail')
+          toastMsg[0].innerText = 'Error occurred. Try again.'
+          showToast(toastMsg[0])
+        })
+      } else if (id.includes('email')) {
+        $(document).ready(function() {
```

Comment:
Why didi you add `$(document).ready()`? It will still work if you will not add that!!

Reply:
This function checks if the document is completely loaded or not. And performs the action only if it is needed. 

YES... there's no need to use this here. 
Thanks :)